_includes/pbc/all/install-features/202410.0/install-dynamic-multistore.md

```diff
@@ -1544,3 +1544,135 @@ docker/sdk boot && docker/sdk up --assets
 - Make sure the store switcher is displayed on the Storefront.
 
 {% endinfo_block %}
+
+## Store in URL
+
+If you want a store name as part of the URL path, follow the steps below.
+
+{% info_block warningBox "Verification" %}
+
+If you are working with pre-generated internal URLs in Twig templates, you must now wrap them using the new Twig function `generatePath()`.
```

Comment:
When is "now"? After installing dynamic multistore?

Reply:
after the release of this functionality